addons/mail/static/src/services/new_message_service/new_message_service_tests.js

```diff
@@ -0,0 +1,68 @@
+/** @odoo-module **/
+
+import { newMessageService } from "./new_message_service";
```

Comment:
I much prefer never using relative import, they are way too annoying to read (checking context becomes important), impossible to grep (to look where a file is potentially imported), and hard to maintain (moving files is a pain).